packages/charts/src/chart_types/heatmap/renderer/canvas/dimensions.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { RgbaTuple } from '../../../../common/color_library_wrappers';
+import { Colors } from '../../../../common/colors';
+import { renderDebugRect } from '../../../../renderers/canvas/utils/debug';
+import { Dimensions } from '../../../../utils/dimensions';
+
+/** @internal */
+export const renderChartContainerDimensions = (ctx: CanvasRenderingContext2D, container: Dimensions) => {
+  const graySemiTransparent = [...(Colors.Gray.rgba.slice(0, 3) as [number, number, number]), 0.5] satisfies RgbaTuple;
+
+  renderDebugRect(
+    ctx,
+    {
+      x: container.left,
+      y: container.top,
+      width: container.width,
+      height: container.height,
+    },
+    0,
+    { color: graySemiTransparent },
+  );
+};
+
+/** @internal */
+export const renderChartDimensions = (ctx: CanvasRenderingContext2D, chart: Dimensions) => {
+  renderDebugRect(
+    ctx,
+    {
+      x: chart.left,
+      y: chart.top,
+      width: chart.width,
+      height: chart.height,
+    },
+    0,
+    { color: Colors.Transparent.rgba },
+    { color: Colors.Red.rgba, width: 1, dash: [4, 4] },
```

Comment:
We should make this thicker to match the other chart types, like [here](https://elastic.github.io/elastic-charts/storybook/?path=/story/debug-options--basic&globals=toggles.showHeader:true;toggles.showChartTitle:false;toggles.showChartDescription:false;toggles.showChartBoundary:false;theme:light&knob-Debug=true&knob-Enable%20debug%20state=true&knob-Grid%20stroke_SmallMultiples%20Styles=1&knob-Horizontal%20inner%20pad_SmallMultiples%20Styles=0.05&knob-Horizontal%20outer%20pad_SmallMultiples%20Styles=0&knob-Persist%20cells%20selection=true&knob-Show%20axes%20panel%20titles_SmallMultiples%20Styles=true&knob-Show%20axes%20title_SmallMultiples%20Styles=true&knob-Vertical%20inner%20pad_SmallMultiples%20Styles=0.1&knob-Vertical%20outer%20pad_SmallMultiples%20Styles=0&knob-categories_Data=4&knob-cell%20density(%)_Data=75&knob-h%20-%20split%20count_Data=2&knob-h%20-%20split_Data=true&knob-v%20-%20split%20count_Data=2&knob-v%20-%20split_Data=true&knob-xScaleType_Data=linear).

```suggestion
    { color: Colors.Red.rgba, width: 4, dash: [4, 4] },
```

Reply:
Done